### PR TITLE
Update resourceManagement.yml with modified message for closed issue

### PR DESCRIPTION
**Pull request by nwojod-MSFT**
Modify the message generated when an issue is closed. Added details to request feedback from developers.

For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

> Summarize the changes, including the goals and reasons for this change. Be sure to call out any technical or behavior changes that reviewers should be aware of.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### Main changes in the PR:

1. <Change 1>
2. <Change 2>

### Validation performed:

1. <Step 1>
2. <Step 2>

### Unit Tests added:

> Unit tests are required for all changes. If no unit tests were added as part of this change, please explain why they aren't necessary.

<Yes/No>

### End-to-end tests added:

<Yes/No>

### Change file added:

> Ensure the change file meets the [formatting requirements](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

<Yes/No>

### Related PRs:

> Remove this section if n/a

### Next/remaining steps:

> List the next or remaining steps in implementing the overall feature in subsequent PRs (or is the feature 100% complete after this?).

> Remove this section if n/a

- [ ] Item 1
- [ ] Item 2

### Screenshots:

> Remove this section if n/a

| Before     | After      |
| ---------- | ---------- |
| < image1 > | < image2 > |

### CODE REVIEW

File: .github/policies/resourceManagement.yml [hunk 1]
```diff
@@ -182,7 +182,7 @@ configuration:
           label: teams-developer-support
       then:
       - addReply:
-          reply: "### Tell us about your experience! \n\nHi ${issueAuthor}! This issue is closed in our system. We would like your [feedback on your experience with our support team and Platform](https://aka.ms/DevSupportFeedback). \n\nBest regards, \nTeams Platform "
+          reply: "### Tell us about your experience! \n\nHi ${issueAuthor}! We've closed this issue in our system, but we'd love to hear your feedback. To let us know how we did, simply use one of the following emoji icons: :thumbsup: for resolved, :thumbsdown: for unresolved, or :heart: for a great experience.. Please use the [link for more detailed feeback](https://aka.ms/DevSupportFeedback). \n\nBest regards, \nTeams Platform "
```
Comment (AE-MS) [inline .github/policies/resourceManagement.yml:185]: >simply use one of the following emoji icons [](http://example.com/codeflow?start=179&length=43)

Which message do you want them to use the icon on? Maybe consider "simply use one of the following emoji icons on this message"

**Review by AE-MS — CHANGES_REQUESTED** [review]
:clock1:

**Review by nwojod-MSFT — COMMENTED** [review]
Updated message to incorporate suggestion.

**Review by jadahiya-MSFT — APPROVED** [review]

**Review by AE-MS — APPROVED** [review]
:shipit:

**Pull request merged**